_meetings/2020-05_EIPvF2F.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: meeting
+name: ELIXIR Interoperability Platform Virtual Face-to-Face 2020
+title: EIPvF2F
+dates: 11-12 May 2020
+start_date: 2020-05-11
+end_date: 2020-05-12
+venue: Virtual
+meeting-name: EIPvF2F
+meeting-url: https://docs.google.com/document/d/1AJvpsXC3X4IIoPhfCYrchdOEX0tfbxkk8YuEeVHBTGY/
+agenda-doc: https://docs.google.com/document/d/1HPz71yj7t4OlEIWOcK3UflXPzpvH7ZRq7RZej33HXow/
+contact-people:
+    - AlasdairGray
+attendees:
```

Comment:
I am not sure attendees is the right term here (I know it is part of the template but maybe something to think about for later). Looking at the resources that will participate, I am sure there will be more attendees from the Bioschemas community. Does it actually refer to facilitators maybe?

Reply:
I've deferred this to an issue [#419](https://github.com/BioSchemas/specifications/issues/419)